packages/api/tests/endpoints/test-executions.js

```diff
@@ -147,21 +179,86 @@ test('executions can be filtered by workflow', async (t) => {
 });
 
 test('GET returns an existing execution', async (t) => {
-  const response = await request(app)
-    .get(`/executions/${fakeExecutions[0].arn}`)
+  const collectionRecord = fakeCollectionRecordFactory();
+  const asyncRecord = fakeAsyncOperationRecordFactory();
+  const parentExecutionRecord = fakeExecutionRecordFactory();
+
+  const executionPgModel = new ExecutionPgModel();
+  const collectionPgModel = new CollectionPgModel();
+  const asyncOperationsPgModel = new AsyncOperationPgModel();
+
+  const [collectionCumulusId] = await collectionPgModel.create(
+    t.context.knex,
+    collectionRecord
+  );
+
+  const [asyncOperationCumulusId] = await asyncOperationsPgModel.create(
+    t.context.knex,
+    asyncRecord
+  );
+
+  const [parentExecutionCumulusId] = await executionPgModel.create(
+    t.context.knex,
+    parentExecutionRecord
+  );
+
+  const executionRecord = await fakeExecutionRecordFactory({
+    async_operation_cumulus_id: asyncOperationCumulusId,
+    collection_cumulus_id: collectionCumulusId,
+    parent_cumulus_id: parentExecutionCumulusId,
+  });
+
+  await executionPgModel.create(
+    t.context.knex,
+    executionRecord
+  );
+  t.teardown(async () => {
+    await executionPgModel.delete(t.context.knex, executionRecord);
+    await executionPgModel.delete(t.context.knex, parentExecutionRecord);
+    await collectionPgModel.delete(t.context.knex, collectionRecord);
+    await asyncOperationsPgModel.delete(t.context.knex, asyncRecord);
+  });
```

Comment:
Does teardown work in reverse order?

Reply:
Interestingly enough, at some point tear down defaults to reverse order, but iirc that's *per block* meaning this shouldn't *need* to be?

https://github.com/nasa/cumulus/pull/2263/files#r646627149

Reply:
I'm not sure I'm following though - *should* it work in reverse order?

Reply:
Huh. I must have read this wrong. What you have here is correct where executions are deleted before collections. For some reason I thought it was reversed.